whatis277/harvest
# Add payment methods table migration

Payment method table with id, user id FK, label, last4, brand, exp month, exp year, is default, created at and updated at

Testing instructions:
1. `dc up postgres -d`
2. `dc run --rm bean tern migrate`
3. `dc run --rm bean tern migrate --destination -1`
4. Ensure no errors
